include/snowflake/client.h

```diff
@@ -570,10 +620,11 @@ snowflake_stmt_get_attr(SF_STMT *sfstmt, SF_STMT_ATTRIBUTE type, void *value);
 /**
  * Executes a statement.
  * @param sfstmt SNOWFLAKE_STMT context.
+ * @param is_put_get_command true this is a put/get command
  *
  * @return 0 if success, otherwise an errno is returned.
  */
-SF_STATUS STDCALL snowflake_execute(SF_STMT *sfstmt);
+SF_STATUS STDCALL snowflake_execute(SF_STMT *sfstmt, sf_bool is_put_get_command);
```

Comment:
This is basically not great addition, because the application has to specify the query type.

Can you add a new snowflake_execute_ex(SF_STMT *sfstmt, sf_bool is_put_get_command)? Then `snowflake_execute` should detect the query type.